.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+- id: jupytext
+  name: jupytext
+  description: Runs jupytext on all notebooks and paired files.
+  language: python
+  entry: jupytext --ignore-unmatched --alert-untracked
+  pass_filenames: true
```

Comment:
@mwouts There is a possible race condition here, maybe? with `require_serial: False` it's possible that `jupytext -sync` is called on both paired files simultaneously in parallel. In practice, this shouldn't cause too many issues, but there could be edge cases I am not considering. `require_serial: True` would prevent the hook from running in parallel, but is probably safer behavior.

Reply:
Great point!

Reply:
Well I must admit that I never thought of that... but you're right, if one instance of Jupytext writes the result at the same time than the other one is starting to read them we will probably run into trouble... So yes I am in favor of `require_serial: True`